CONTRIBUTING.md

```diff
@@ -307,6 +307,15 @@ Run:
 docker-compose up
 ```
 
+
+### Creating a New User
```

Comment:
Could you please move this section down, under the "Shared" header? Line 326 looks like a good spot.

Reply:
Sure! I will do that.

Reply:
👍